### ISSUE TITLE
choledocholithiasis

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11068]
**Preferred term label:**
choledocholithiasis

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of gallstones within the common bile duct.

**Parent term (use hpo.jax.org/app)**
Abnormal extrahepatic bile duct morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding new term: 
- ID: HP:6001269
- label: Choledocholithiasis
- parents: [Abnormal extrahepatic bile duct morphology (HP:0035013)](https://hpo.jax.org/app/browse/term/HP:0035013)
- pmid: PMID:28722990
- def: Choledocholithiasis is the presence of stones within the common bile duct.
- comment: It is estimated that common bile duct stones are present in anywhere from 1-15% of patients with cholelithiasis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11068
- synonyms:
